src/Object/Api/Twitter/User.php

```diff
@@ -106,16 +119,16 @@ public function __construct(array $publicContact, array $apcontact = [], array $
 			unset($this->entities);
 		}
 		$this->description             = BBCode::toPlaintext($publicContact['about']);
-		$this->profile_image_url_https = $userContact['avatar'] ?? $publicContact['avatar'];
+		$this->profile_image_url_https = Contact::getAvatarUrlForUrl($publicContact['url'], $uid, Proxy::SIZE_MICRO);
 		$this->protected               = false;
 		$this->followers_count         = $apcontact['followers_count'] ?? 0;
 		$this->friends_count           = $apcontact['following_count'] ?? 0;
 		$this->listed_count            = 0;
 		$this->created_at              = api_date($publicContact['created']);
 		$this->favourites_count        = 0;
-		$this->verified                = false;
+		$this->verified                = $uid != 0;
 		$this->statuses_count          = $apcontact['statuses_count'] ?? 0;
-		$this->profile_banner_url      = '';
+		$this->profile_banner_url      = Contact::getHeaderUrlForId($publicContact['id'], '', $publicContact['updated']);
```

Comment:
same as above, but you could additionally introduce `publicContact['profile_banner`]` for example.